docker-compose.yml

```diff
@@ -47,6 +47,16 @@ services:
       - CLOUDFLARE_ACCESS_KEY_ID
       - CLOUDFLARE_SECRET_ACCESS_KEY
       - CLOUDFLARE_BUCKET
+      - POSTMARK_MAILER_FEATURE
+      - POSTMARK_SERVER_API_TOKEN
+      - POSTMARK_FROM_EMAIL
+      - POSTMARK_PASSWORD_RESET_TEMPLATE_ALIAS
+      - ACTION_MAILER_HOST
+      - PRODUCT_NAME
+      - SUPPORT_URL
+      - COMPANY_NAME
+      - COMPANY_ADDRESS
+      - PASSWORD_RESET_TOKEN_TTL_MINUTES
```

Comment:
Let's order all of the ENV keys in this whole block so that it matches the order of the file in options.yml, e.g. POSTMARK FEATURE would be up with the other features. Feel free to reorder other ones, I haven't checked it in awhile.

Every so often I compare options.yml to this file to see if we've missed any and that comparison is easy when they're ordered identically but it's hard when they're ordered/grouped differently.